tests/integration/conftest.py

```diff
@@ -157,12 +158,18 @@ def _cleanup(
         batch_setup.teardown()
 
 
+@pytest.fixture(scope="session")
+def _context() -> AbstractDataContext:
+    return gx.get_context(mode="ephemeral")
```

Comment:
Me and @joshua-stauffer observed that the context was being re-created and data sources were not being saved for use in the test. We could also scope this down to the `module`. Maybe there is a better solution that involves changing the order in which the fixtures are loaded?

Reply:
yeah, something seems off here. I wonder if `ProjectManager.set_context` is being called somewhere else during test execution, and its overriding the context.

Reply:
`set_context` in `multi_source_batch` fixed it